packages/pentaho/src/Canvas/SidePanel/SidePanel.tsx

```diff
@@ -108,6 +108,10 @@ export const HvCanvasSidePanel = forwardRef<
     tabs?.[0]?.id ?? "none",
   );
 
+  useEffect(() => {
+    handleSidePanelOpen?.(open);
```

Comment:
can we remove the eslint-disable below? it's exactly the same use-case

Reply:
Updated the PR.